### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by blonkm** [gottcode/cutemaze#2]
I notice moving the mouse through the maze is slow. I have a Windows 7, dual core, 5GB machine. I think a simple maze program should run fast. Is it inentionally slow? Even without the smooth movement setting, the step-by-step-approach is annoying. This is the first time I use a QT program. Is Qt slow in general?

**Comment by gottcode** [comment]
No, it isn't intentionally slow, and nor is it Qt's fault. CuteMaze runs quite fast on my Linux installs, but sometimes it runs slowly on Windows for me. I have not yet figured out why, but I am looking into it.
